src/main/java/frc/robot/components/TrigonXboxController.java

```diff
@@ -106,13 +111,31 @@ public void intermittentRumble(int quantity) {
     }
 
     @Override
-    public double getRightY() {
-        return -super.getRightY();
+    public double getLeftX() {
+        if(super.getLeftX() < deadband && super.getLeftX() > -deadband)
+            return 0;
+        return squared ? Math.signedSquare(super.getLeftX()) : super.getLeftX();
     }
 
     @Override
     public double getLeftY() {
-        return -super.getLeftY();
+        if(super.getLeftY() < deadband && super.getLeftY() > -deadband)
+            return 0;
+        return squared ? Math.signedSquare(super.getLeftY()) : super.getLeftY();
+    }
+
+    @Override
+    public double getRightX() {
+        if(super.getRightX() < deadband && super.getRightX() > -deadband)
+            return 0;
+        return squared ? Math.signedSquare(super.getRightX()) : super.getRightX();
+    }
+
+    @Override
+    public double getRightY() {
+        if(super.getRightY() < deadband && super.getRightY() > -deadband)
+            return 0;
+        return squared ? Math.signedSquare(super.getRightY()) : super.getRightY();
     }
```

Comment:
make a deadband function and a squared function and have these functions returned something like 
`processedValue(super.getRightY())`